angular/test/protractor/helpers/page_helper.coffee

```diff
@@ -40,9 +40,6 @@ module.exports = new class PageHelper
   findFirst: (selector) ->
     element.all(By.css(selector)).first()
 
-  clickLast: (selector) ->
```

Comment:
I think we still need this; I can see a couple tests which still use this function